doc/decisions/1_in_discussion/spec_expressiveness.md

```diff
@@ -2,22 +2,61 @@
 
 ## Problem
 
-Currently, you can easily come into wrong assumptions that something would work in the specification.
+The current implementation of the spec plugin is copying metadata to all other namespaces.
+Metadata e.g. `type` or `description` describes a certain key.
 
-We need to find minimal requirements to implement a sane spec plugin.
+Sometimes there are keys with similar or identical specifications.
+We want to define those in a single place and not have to manually copy metadata.
+`Globbing` expressions in key names comes into place at that time. 
+By providing certain wildcard characters e.g. `_` or `#` one can match multiple keys.
+This removes the need to explicitly define specification for keys.
```

Comment:
Weird order. I'd start with the "Why?" then explain the "How?". So first give an example of specifications and then explain that the current `spec` achieves this by copying metadata.

You should probably also start this without mentioning globbing. A specification without globbing still makes sense and is easier to understand. In your example you could for example have

```ni
[public/ip]
meta:/default = 127.0.0.1
meta:/description = "The IP via which this application is accessible."

[public/port]
meta:/default = 8080
meta:/description = "The port on which this application is accessible."
```